From c603640b2a31259991eb15b563521a856d17abb1 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Wed, 03 Jul 2019 14:38:28 +0300
Subject: [PATCH] [SW-4400] linux: goya: gaudi: add support for i2c-slave-eeprom

Add support in kernel for i2c-slave-eeprom driver
which adds support to all i2c-slave requests
for goya and gaudi devices.

Change-Id: I144c0e4ab620e89ab32d444948ba43131cfed168
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/board/habanalabs/gaudi/linux_asic.fragment b/board/habanalabs/gaudi/linux_asic.fragment
index 9946ee8..d4c5042 100644
--- a/board/habanalabs/gaudi/linux_asic.fragment
+++ b/board/habanalabs/gaudi/linux_asic.fragment
@@ -28,3 +28,4 @@
 CONFIG_POSIX_MQUEUE=y
 CONFIG_I2C_MUX=y
 CONFIG_I2C_MUX_PCA9541=y
+CONFIG_I2C_SLAVE_EEPROM=y
diff --git a/board/habanalabs/goya/linux_asic.fragment b/board/habanalabs/goya/linux_asic.fragment
index 9ec130c..10fef31 100644
--- a/board/habanalabs/goya/linux_asic.fragment
+++ b/board/habanalabs/goya/linux_asic.fragment
@@ -27,3 +27,4 @@
 CONFIG_SENSORS_TMP401=y
 CONFIG_JFFS2_FS=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_I2C_SLAVE_EEPROM=y
